install.sh

```diff
@@ -4,16 +4,20 @@
 # Installs nodenv-default-packages under $PREFIX.
 
 set -e
-set -u
-
-cd "$(dirname "$0")"
 
 if [ -z "${PREFIX}" ]; then
   PREFIX="/usr/local"
 fi
 
+set -u
```

Comment:
Mind moving these back up?

The PREFIX defaulting can be done via:
`: "${PREFIX:=/usr/local}"` to avoid the set -u issue.